target/linux/ramips/image/mt7620.mk

```diff
@@ -909,6 +909,20 @@ define Device/tplink_archer-c2-v1
 endef
 TARGET_DEVICES += tplink_archer-c2-v1
 
+define Device/tplink_archer-c5-v4
+  $(Device/Archer)
```

Comment:
I've recently changed the shared definitions for tplink in ramips:
https://github.com/openwrt/openwrt/commit/3227afbbabf06ac68a27bb5ad68c1928a06c4203

You will need to replace Device/Archer by Device/tplink-v2 here, add "SOC := mt7620a", and remove "IMAGES += factory.bin".